### ISSUE TITLE
[Bug] Accessibility Name and HelpText for Label visible in Android UI

### ISSUE THREAD

**Issue by Poli97** [xamarin/Xamarin.Forms#9707]
### Description
Accessibility values added on empty Label are visible in Android UI
Same as [this issue](https://github.com/xamarin/Xamarin.Forms/issues/3208), but with Label. 

### Steps to Reproduce

1. Create a Forms project
2. Add an empty Label, with a background and a certain size, to the Layout
3. Set the label in accessible tree with `AutomationProperties.SetIsInAccessibleTree(label, true)`
4. Set an accessibility name to it with `AutomationProperties.SetName(label, "Test accessibility name")` 
5. Run the app in emulator or physical device

CODE SAMPLE:
C#
```
Label label = new Label
            {
                BackgroundColor = Color.Orange,
            };
            AutomationProperties.SetIsInAccessibleTree(label, true);
            AutomationProperties.SetName(label, "Test accessibility name");
            AbsoluteLayout.SetLayoutBounds(label, new Rectangle(0.5, 0.5, 0.3, 0.2));
            AbsoluteLayout.SetLayoutFlags(label, AbsoluteLayoutFlags.All);

            absolutelayout.Children.Add(label);`
```

### Expected Behavior
An empty label with no text should be displayed

### Actual Behavior
In the label it is shown the accessibility text set with `AutomationProperties.SetName(label, "Test accessibility name")`, in this case "Test accessibility name" is shown. (In native Android `ContentDescription` works fine and doesn't show any text).

### Basic Information

- Version with issue: All the latest tried, 4.3 and 4.4 and also pre release 4.5 pre 4
- Last known good version: none
- IDE: Visual Studio for Mac 8.4.6 (build 36)
- Platform Target Frameworks: 
  - Android: 9.0, API 28
- Nuget Packages: Xamarin.Essentials 1.3.1 and 1.4.0
- Affected Devices: Seems on all android devices

### Screenshots
![image](https://user-images.githubusercontent.com/58602175/75241914-5b4b6e80-57c7-11ea-8d54-f675db1a9026.png)

### Reproduction Link

[Git repo](https://github.com/Poli97/AccessibilityName)

**Comment by PureWeen** [comment]
Please try SemanticEffect on the XCT https://github.com/xamarin/XamarinCommunityToolkit/pull/1240

The behavior of `SemanticEffect` is much more accurate
